canvas-script.js

```diff
@@ -198,4 +202,17 @@ function renderPosts(posts) {
 
 function openUp() {
   top5Container.style.display = "block";
+}
+
+export function stopGame() {
```

Comment:
When the game stops:
1. Hide canvas
2. Clear canvas
3. Clear the "ball array" -- amount of balls on screen.